src/t_stream.c

```diff
@@ -127,20 +127,29 @@ unsigned char *lpReplaceInteger(unsigned char *lp, unsigned char **pos, int64_t
 /* This is a wrapper function for lpGet() to directly get an integer value
  * from the listpack (that may store numbers as a string), converting
  * the string if needed. */
-int64_t lpGetInteger(unsigned char *ele) {
+static inline int64_t lpGetIntegerIfValid(unsigned char *ele, int *out_valid) {
```

Comment:
/s/out_valid/is_valid?

Reply:
i meant to note that it's an output argument.

Reply:
Oh, now I get it, I thought it meant if the output was valid. I can believe that's a thing.